ixwebsocket/IXWebSocketTransport.cpp

```diff
@@ -242,9 +245,19 @@ namespace ix
         return now - _lastReceivePongTimePoint > std::chrono::seconds(_pingTimeoutSecs);
     }
 
-    void WebSocketTransport::poll()
+    bool WebSocketTransport::closingDelayExceeded()
     {
-        PollResultType pollResult = _socket->poll(_pingIntervalOrTimeoutGCDSecs);
+        std::lock_guard<std::mutex> lock(_closingTimePointMutex);
+        auto now = std::chrono::steady_clock::now();
+        return now - _closingTimePoint > std::chrono::milliseconds(kClosingMaximumWaitingDelayInMs);
+    }
+
+    WebSocketTransport::PollPostTreatment WebSocketTransport::poll()
+    {
+        // we need to have no timeout if state is CLOSING
```

Comment:
agreed, good comment.

Reply:
the name of the variable is inconsistent with the rest of the code / should be timeoutDealySecs; Or just timeout, or the variable used in the ::poll signature.

Reply:
To be changed yes